src/chigame/games/views.py

```diff
@@ -7,3 +8,17 @@ class GameListView(ListView):
     model = Game
     queryset = Game.objects.all()
     template_name = "games/game_list.html"
+
+
+class GameCreateView(CreateView):
+    model = Game
+    fields = ["name", "description", "min_players", "max_players"]
+    template_name = "games/game_form.html"
+    success_url = reverse_lazy("game-list")
+
+
+class GameEditView(UpdateView):
+    model = Game
+    fields = ["name", "description", "min_players", "max_players"]
+    template_name = "games/game_form.html"
+    success_url = reverse_lazy("game-list")
```

Comment:
Once #71 is merged, change this to the detail view for a game. Since then we're giving the user feedback on how their requested action was processed. 
Since with the code as it is, if I didn't change the title of a game I wouldn't know if my changes were successful without manually going to the detail view.